### ISSUE TITLE
Does not work with .NET Core

### ISSUE THREAD

**Issue by kspearrin** [bitpay/csharp-bitpay-client#34]
Is this library suppose to be usable?

1. Created new console app
2. Installed BitPay SDK from nuget.
3. 

    class Program
    {
        static void Main(string[] args)
        {
            var bp = new BitPay();
            Console.ReadLine();
        }
    }

4. Run program.

Result:

```
System.IO.FileNotFoundException: 'Could not load file or assembly 'BitCoinSharp, Version=0.2.199.60, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.'
```

Doesn't seem to work at all....

**Comment by danescsvn** [comment]
@kspearrin thank you for the feedback. We will look into this issue and provide an update soon. Thank you.

**Comment by Kukks** [comment]
Just use the unofficial implementation of this api @ https://github.com/MetacoSA/NBitpayClient

**Comment by kspearrin** [comment]
@Kukks Yes, this is what I have resorted to since this one is not usable in its current state.

**Comment by anbugal** [comment]
This issue is been fixed in the new version v2.0.1904
